src/main/java/graphql/execution/ExecutionContext.java

```diff
@@ -43,7 +42,7 @@ public class ExecutionContext {
     private final ImmutableMap<String, FragmentDefinition> fragmentsByName;
     private final OperationDefinition operationDefinition;
     private final Document document;
-    private final ImmutableMapWithNullValues<String, Object> variables;
+    private final CoercedVariables coercedVariables;
```

Comment:
`ExecutionContext` can only contain coerced variables. This change makes it clearer